presto-tpcds/src/test/java/com/facebook/presto/tpcds/statistics/TestTpcdsLocalStats.java

```diff
@@ -40,6 +41,7 @@ public void setUp()
         Session defaultSession = testSessionBuilder()
                 .setCatalog("tpcds")
                 .setSchema("sf1")
+                .setSystemProperty(PRINT_STATS_FOR_NON_JOIN_QUERY, "true")
```

Comment:
I guess you wanted to put that in the previous commit

Reply:
Good  catch.  This comes with the second commit b/c it is associated boolean explain parameter, caller is  
  LogicalQueryRunner.java#848:  LogicalPlanner logicalPlanner = new LogicalPlanner(wrappedStatement instanceof Explain....